contrib/builder/rpm/fedora-22/Dockerfile

```diff
@@ -7,7 +7,7 @@ FROM fedora:22
 RUN dnf install -y @development-tools fedora-packager
 RUN dnf install -y btrfs-progs-devel device-mapper-devel glibc-static libseccomp-devel libselinux-devel libtool-ltdl-devel selinux-policy selinux-policy-devel sqlite-devel tar
 
-ENV SECCOMP_VERSION v2.2.3
+ENV SECCOMP_VERSION 2.2.3
```

Comment:
rebased*

On Tuesday, January 19, 2016, Jessie Frazelle <jess@docker.com> wrote:

> Oh ya I debased this on the other one for testing ;)
>
> On Tuesday, January 19, 2016, Tianon Gravi <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> In contrib/builder/rpm/fedora-22/Dockerfile
>> <https://github.com/docker/docker/pull/19378#discussion_r50204437>:
>>
>> > @@ -7,7 +7,7 @@ FROM fedora:22
>> >  RUN dnf install -y @development-tools fedora-packager
>> >  RUN dnf install -y btrfs-progs-devel device-mapper-devel glibc-static libseccomp-devel libselinux-devel libtool-ltdl-devel selinux-policy selinux-policy-devel sqlite-devel tar
>> >
>> > -ENV SECCOMP_VERSION v2.2.3
>> > +ENV SECCOMP_VERSION 2.2.3
>>
>> Yeah, this is good -- she updated the lines below to match the top-level
>> Dockerfile too, which was the issue. [image: :+1:]
>>
>> (This could actually be a separate PR and it'd be merged already. [image:
>> :smile:])
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/docker/docker/pull/19378/files#r50204437>.
>>
>